ecosystem/indexer-server/typescript/src/api/v1/openapi.yaml

```diff
@@ -1155,7 +1155,7 @@ components:
     AccountResource:
       description: Account resource is a Move struct value belongs to an account.
       example:
-        type: 0x1::AptosAccount::Balance<0x1::XDX::XDX>
+        type: 0x1::coin::CoinStore<0x1::XDX::XDX>
```

Comment:
This file is supposed to be autogenerated and should use the other spec

Reply:
Are there instructions or regenerating?

Reply:
The new spec has none of this, so once the old one is deleted you don't have to do anything!

Reply:
this is autogenerated but still checked into the repo though. I don't see it being updated automatically when running build or lint

Reply:
It's outside the scope of this.  It's supposed to regenerate automatically or tell you in the CI if it wasn't updated.

Either way the currency code isn't even in the spec to be generated so there is no change in the autogenerated code.